### ISSUE TITLE
Ui Designer - long press not showing pop up menu - Bug

### ISSUE THREAD

**Issue by Phuket2** [omz/Pythonista-Issues#512]
If you long press on a object in the ui Designer and say select copy. Then if you long press somewhere on the sheet the pop up menu does not come up so you can paste.  
I am on beta 311017 with ios 11.2.2
Would be nice if someone else could confirm this

**Comment by omz** [comment]
I can’t reproduce this. If a UI element was previously copied, the “Paste” menu does appear here (after long press).

**Comment by zrzka** [comment]
@omz no matter what I do, there's no paste / menu. Nothing for simple tap, animated circle for long press, ... https://www.dropbox.com/s/5uts5gpzy4817kj/no-paste-in-designer.mov?dl=0

**Comment by zrzka** [comment]
Same on iPhone SE.

**Comment by omz** [comment]
Hmm, that’s odd. Does this also happen when you launch Pythonista in safe mode (no `pythonista_startup`)?

**Comment by omz** [comment]
It works fine for me both on the iPhone SE and iPad Pro.

**Comment by zrzka** [comment]
Yes, I don't have nothing fancy like BM on iPhone. Freshly installed, nothing there. Happens even when I launch Pythonista via Safari. I have it on iPhone just for testing cases like this one.

**Comment by omz** [comment]
I’ve just seen that it doesn’t work quite right here either. The menu disappears too quickly, but it’s still possible to use it (it appears briefly, then disappears).

**Comment by zrzka** [comment]
That's weird. Made another couple of recordings and did go through them frame by frame in FCP to see if the menu is there even for a short period of time and I'm blind :) No luck. Nothing or just animated blue circle. Playing with pencil, same behaviour, ... On the other side, can paste attributes to other control, so, the control is copied to the pasteboard for sure, correct attributes are pasted.

**Comment by Phuket2** [comment]
I am not sure what version it changed, but I was getting the menu up and it also disappeared very quickly.  But if you tried a little harder you could get the menu to activate. But now, i dont see any flashes at all.  I have tried pretty hard to get it to show, but I can not.
@zrzka, I normally never upgrade to the store version, only because I dont know what happens to TestFlight. I was not sure if a beta came out on test flight if I would be able to overwrite the store version or not. So I guess what you are saying, its ok to upgade to the store version and then when a new beta is released it will overwrite the store version and keep all your info intact?

**Comment by zrzka** [comment]
@Phuket2 I update to store & TF & store ... even several times / day when I'm testing new things. All data intact. But better backup -> much easier these days with iCloud.

**Comment by idchlife** [comment]
No sight of a menu for me. I actually thought before finding this issue that Pythonista does not even have this functionality. No copy paste elements in UI designer for me 😢

**Comment by johnridesabike** [comment]
This has been affecting me as well for the past month on iPad and iPhone SE.

Occasionally the paste button will appear on long press, but almost always it won't. I haven't been able to notice any pattern to it.

One workaround that I've found is that if you create a subview of the same type that you want to paste (e.g. Button, Image, etc), you can select that and "paste attributes." It's not exactly the same, but it's better than nothing.

**Comment by bdalton12** [comment]
I ended up modifying the pyui file in a text editor.  Good to know the "paste attributes" also works.
